packages/workspace/src/schematics/preset/preset.ts

```diff
@@ -93,32 +93,49 @@ function createPreset(options: Schema): Rule {
         },
         { interactive: false }
       ),
-      schematic(
-        'library',
-        { name: 'api-interface', framework: 'none' },
-        { interactive: false }
-      ),
+      schematic('library', { name: 'api-interfaces' }, { interactive: false }),
       setDefaultCollection('@nrwl/angular'),
-      connectFrontendAndApi(options)
+      connectAngularAndNest(options)
     ]);
   } else if (options.preset === 'react-express') {
-    throw new Error(`Not implemented yet`);
+    return chain([
+      externalSchematic(
+        '@nrwl/react',
+        'application',
+        {
+          name: options.name,
+          style: options.style,
+          babel: true,
+          linter
+        },
+        { interactive: false }
+      ),
+      externalSchematic(
+        '@nrwl/express',
+        'application',
+        {
+          name: 'api',
+          frontendProject: options.name
+        },
+        { interactive: false }
+      ),
+      schematic('library', { name: 'api-interfaces' }, { interactive: false }),
+      setDefaultCollection('@nrwl/react'),
+      setDefaultLinter(linter),
+      connectReactAndExpress(options)
+    ]);
   } else {
     throw new Error(`Invalid preset ${options.preset}`);
   }
 }
 
-function connectFrontendAndApi(options: Schema) {
+function connectAngularAndNest(options: Schema) {
```

Comment:
I would go with connectAngularAndApi as there is no real reason why it have to be Nest and not express.